.travis.yml

```diff
@@ -11,7 +10,7 @@ script:
 - go test -v $(go list ./... | grep -v /vendor/)
 - go vet $(go list ./... | grep -v /vendor/)
 - diff <(goimports -d $(find . -type f -name '*.go' -not -path "./vendor/*")) <(printf "")
-- for d in $(go list ./... | grep -v /vendor/); do diff <(golint $d) <(printf ""); done
+#- for d in $(go list ./... | grep -v /vendor/); do diff <(golint $d) <(printf ""); done
```

Comment:
I think we should quickly document the missing things and get lint clean before merging